src/scripts/Dockerfile.android

```diff
@@ -0,0 +1,20 @@
+FROM debian:stretch AS debian
+ARG ANDORID_SDK_VER
+ARG ANDROID_ARCH
+ARG ANDROID_TOOLCHAIN_SUF
+RUN apt-get update && apt-get -y --no-install-recommends install \
+	make \
+	python3-sphinx
```

Comment:
Similarly I don't think we need Sphinx here, that's what the `docs` Travis build is for.